greenberry/gbtest/test.py

```diff
@@ -21,5 +21,15 @@ def test_null(self):
         self.assertTrue(error)
 
 
+    def test_eval_add(self):
+        x = 'print eval 1+2'
+        temp_stdout = StringIO()
+        # redirect stdout to catch print statement from eval function
+        with contextlib.redirect_stdout(temp_stdout):
```

Comment:
can you add stdout capture as a separate function?

Reply:
yeah sure!